gradle/versions.gradle

```diff
@@ -89,6 +89,6 @@ dependencyManagement {
 
     dependency 'org.xerial.snappy:snappy-java:1.1.7.3'
 
-    dependency "tech.pegasys.pantheon:plugin-api:1.1.3"
+    dependency "tech.pegasys.pantheon:plugin-api:1.2.0-SNAPSHOT"
```

Comment:
We can't depend on snapshot versions.  This will need to wait until we get the plugin-api version updated.